lib/pages/chat_draft/draft_chat_view.dart

```diff
@@ -67,196 +67,210 @@ class DraftChatView extends StatelessWidget {
-                          margin: const EdgeInsetsDirectional.only(end: 8.0),
-                          decoration: BoxDecoration(
-                            borderRadius:
-                                const BorderRadius.all(Radius.circular(25)),
-                            color: Theme.of(context).colorScheme.surface,
-                          ),
-                          child: Row(
-                            children: [
-                              Expanded(
-                                child: InputBar(
-                                  minLines: 1,
-                                  maxLines: 8,
-                                  autofocus: !PlatformInfos.isMobile,
-                                  keyboardType: TextInputType.multiline,
-                                  textInputAction: AppConfig.sendOnEnter
-                                      ? TextInputAction.send
-                                      : null,
-                                  onSubmitted: controller.onInputBarSubmitted,
-                                  focusNode: controller.inputFocus,
-                                  controller: controller.sendController,
-                                  decoration: InputDecoration(
-                                    hintText: L10n.of(context)!.chatMessage,
-                                    hintMaxLines: 1,
-                                    hintStyle: Theme.of(context)
-                                        .textTheme
-                                        .bodyLarge
-                                        ?.merge(
-                                          Theme.of(context)
-                                              .inputDecorationTheme
-                                              .hintStyle,
-                                        )
-                                        .copyWith(letterSpacing: -0.15),
-                                  ),
-                                  onChanged: controller.onInputBarChanged,
+                Expanded(
+                  child: Center(
+                    child: DirectDraftChatView(
+                      onTap: () => controller.inputFocus.requestFocus(),
+                    ),
+                  ),
+                ),
+                Column(
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.only(bottom: 8.0),
+                      child: Row(
+                        crossAxisAlignment: CrossAxisAlignment.center,
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        children: [
+                          TwakeIconButton(
+                            tooltip: L10n.of(context)!.more,
+                            margin: const EdgeInsets.only(right: 4.0),
+                            icon: Icons.add_circle_outline,
+                            onPressed: () {
+                              controller.showImagesPickerBottomSheetAction(
```

Comment:
refactor: Should call controller without onItemAction and onSendTap, because it call controller again